### DISCUSSION TITLE
Better Documentation for figure customization

### DISCUSSION THREAD

**Discussion by tomas-brito-devoto** [quarto-dev/quarto-cli#3492] [Q&A] [answered]
Hi! 
I've been trying to include a .svg figure into a .md document and struggled to set the rendering width option! I won't tell how much it took to actually solve it, but I ended up posting it in the [RStudio Community Forum.](https://community.rstudio.com/t/scale-svg-file-in-quarto-msword-report/153841) As per recommended by the moderator, I'm posting the solution here with a request to update the Quarto documentation to better explain how to customize several image options in the source view.

Personally, I struggled with specifying the width option, but I wonder how many other options are available and how multiple ones can be specified for one image (should I separate multiple options with a comma, space, etc.). 

My solution:
`![image description](./path/myimage.svg){#fig-ref width="170mm"} (px, mm, cm, in, and % are accepted)`

Also, images are rendered into MSWord with an 'in line with text' layout. Is there a way to insert images with other layouts? Say, an image left of the text.  

Thanks!

ps: This is my first post and I'm not sure if this is the correct category for this request.

**Comment by mcanouil** [comment, marked as answer]
The specification follows HTML syntax and allows any (or almost) HTML key/value attribute.
Most of the options come from Pandoc, i.e., it's not a Quarto specificity.
See <https://pandoc.org/MANUAL.html#images>.

Regarding Word docx, this format is not the best, Quarto improved some things by adding some steps before giving the results to Pandoc, but it's far from perfect.
I can't tell you if this is something Quarto team can improve or if it is something tied up to Pandoc.

**Comment by tomas-brito-devoto** [comment]
Thanks for your reply! I wasn't aware of the HTML syntax of pandoc. My bad I guess, so thanks for the clarification. It'll come in handy for futrue troubleshooting. 

Regarding Word, I do hope both Quarto and Pandoc continue to improve compatibilty.
